### ISSUE TITLE
Syncthing - /rest/system/upgrade Error 500

### ISSUE THREAD

**Issue by sgtpepperaut** [turnkeylinux/tracker#1583]
This happens on virgin installs. Any idea what that could be?

![image](https://user-images.githubusercontent.com/14684675/111624644-3d92e200-87ec-11eb-8182-17435ad58867.png)

**Comment by JedMeister** [comment]
Firstly, in future, please provide MUCH more detail when reporting problems. It can not be overstated that when reporting bugs, there is no such thing as too much information! The more info the better! If the info is irrelevant, then it can be skipped over, but not providing enough info will result in wasted time and energy, and/or delay he process (e.g. asking for more info).

As a bare minimum, it would have been useful if you had explicitly noted where you saw this error message, whether or not you are experiencing any other issues that may be related (e.g. whether it causes any problems or is just an error message). It's also worth including the exact version you are using (if you don't know, you can run `turnkey-version`) and where it is running (e.g. ISO install to bare metal; OVA install to VirtualBox; AWS EC2 launched via the Hub) and the system resources the server has (whether virtual or real).

For reference, I wasted considerable time trying to reproduce this issue as due to the lack of info you provided, it wasn't immediately apparent to me that this was an error in the browser dev tools console. Once it occurred to me that your screenshot was from the browser dev tools console, then I found it in 2 seconds....

Anyway, unless you are actually experiencing any problems, my gut feeling is that this is just noise (i.e. technically an error, but not actually a problem). But I'm unsure, so I've opened an [issue with Syncthing themselves](https://github.com/syncthing/syncthing/issues/7504). Note how much info I provided (obviously you may not have known as much about the config as me; but you could have shared much more info than you did).

FYI I have confirmed the error message but can't find any fault in Syncthing's operation. I tested this issue using the default TurnKey Syncthing appliance. Plus I also updated to the latest Syncthing release (`apt update && apt install syncthing`). I also tested using default config (i.e. not behind an Nginx proxy) as well and it's appearance is consistent. Also, my internet searches bought up 2 results directly relating to Syncthing, both where that was noted, along with other specific problems and there was no comment on the message. This reinforces my suspicion that it's just noise...

Anyway, sorry if my response was a bit ranty. I do genuinely appreciate TurnKey users providing feedback, and especially bug reports. But we're a small team, so wasted time is incredibly frustrating...

**Comment by sgtpepperaut** [comment]
I'll do better next time! It honestly didn't occur to me that anybody in here doesn't recognize the debug console..but we all live in our bubble. Thanks for you time and work.

**Comment by JedMeister** [comment]
All good @sgtpepperaut - Apologies if I seemed a bit ranty! :smile:

> It honestly didn't occur to me that anybody in here doesn't recognize the debug console..but we all live in our bubble.

I'm much more of a sysadmin than anything else, so actually don't use the dev console much. Once I realised, it was obvious and I'll likely not make that mistake again. But yeah, you're right about us all in our own bubble... Curse of knowledge and all that! :+1:  

Regardless though, other than the error message in the console are you noticing any issues with Syncthing? FWIW, I also noticed another Angular "violation". But as I say, I couldn't actually fault Syncthing's operation...

FWIW, I've spoken with a colleague and he noted that "javascript is the bash of the online world" - i.e. a powerful language that can do some cools things, but doesn't really enforce any specific structure, so allows users/developers to do things in messy ways... As such these sort of (javascript) framework errors are (perhaps surprisingly) common.
